File: app/src/main/java/com/example/android/moviesremake/realm/MovieRealm.java
package com.example.android.moviesremake.realm;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by PepovPC on 1/14/2018.
 */

public class MovieRealm extends RealmObject {



    @PrimaryKey
    private String id;

    private String image;
    private int movieID;
    private String release;
    private String overview;
    private String vote;


    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public int getMovieID() {
        return movieID;
    }

    public void setMovieID(int movieID) {
        this.movieID = movieID;
    }

    public String getRelease() {
        return release;
    }

    public void setRelease(String release) {
        this.release = release;
    }

    public String getOverview() {
        return overview;
    }

    public void setOverview(String overview) {
        this.overview = overview;
    }

    public String getVote() {
        return vote;
    }

    public void setVote(String vote) {
        this.vote = vote;
    }
}

File: app/src/main/java/com/example/android/moviesremake/realm/MovieRealm.kt
package com.example.android.moviesremake.realm

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

/**
 * Created by PepovPC on 1/14/2018.
 */

open class MovieRealm : RealmObject() {


    @PrimaryKey
    var id: String? = null

    var image: String? = null
    var movieID: Int = 0
    var release: String? = null
    var overview: String? = null
    var vote: String? = null
}
